astroplan/plots/mplstyles.py

```diff
@@ -0,0 +1,92 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+from astropy.visualization import astropy_mpl_style
+
+__all__ = ['light_style_sheet', 'dark_style_sheet',
+           'available_style_sheets']
+
+available_styles = ["light_style_sheet", "dark_style_sheet"]
+
+def available_style_sheets():
```

Comment:
@bmorris3 – No. This is a known issue for the Sphinx code in astropy-helper https://github.com/astropy/astropy-helpers/issues/117 . Module-level variables don't show up in the generated API docs, so you can't link to them.